### ISSUE TITLE
how can I solve that

### ISSUE THREAD

**Issue by Viniciusalgueiro** [TaigaOsguthorpe/Maki-Bot#3]
I wondered if anyone could help me, I wanted to use the bot to rank the list of people who interacted the most on the channel, either by permanence or by chat, but I didn't find a command for that. would have any command to help me?

**Comment by TaigaOsguthorpe** [comment]
Hey @Viniciusalgueiro . I don't maintain this project anymore, I generally use GitHub as a dumping ground of my code for further reference for me down the line and maybe to help one or two other people figure something out that took me a while.

The code currently has no ability to rank people by how much they have tallked in the text chat, bots from when I used to use discord's API only had about 2 weeks of previous text to be able to go through so it would have to the ranking process from when the bot is enabled + the two weeks of data it had allready.

Maki Bot's code is most likley very out of date now. If you are studying to make your own Discord bot to do this type of command then I don't know if my bot's source code is a very good place to start as it uses out of date techniques and has some beginer mistakes commented out but probably a lot more in there since I have not touched the source code in over 2 years now.
However, commands for this bot can be written and slotted in with relitive ease after you know how commands are initialized. But I recomend you to write your own bot from the ground up as it will most likley be easier than learning my programs code.

If you are interested in making your own discord bot or basing one off of mine then this bot was written in and for [Python ](https://www.python.org/) 3.5 - 3.6 using the (at the time called) rewrite branch of [discord.py](https://github.com/Rapptz/discord.py) ([API references documentation here](https://discordpy.readthedocs.io/en/latest/api.html)) and therefore making use of [asyncio](https://docs.python.org/3/library/asyncio.html). I don't use discord.py anymore so it may have changed but from a quick glance it seems the then called rewrite branch is now the main branch of the code and the prefered api. If you do use any of my code please adhear to the MIT licence it has as I figured it out from the ground up without any other src to look at back when I first developed it in late 2017.

I took a quick look at your GitHub profile and you seem to primarally do frontend development so I am unsure on how firmilar you will be with this type of an aplication and especially [asynchrony ](https://en.wikipedia.org/wiki/Asynchrony_(computer_programming)) in programing. I recomend you start into asyncio for python and discord.py to get a good understanding of what is going on as I most likley won't ever be devloping this bot again. The most activity I may do on this repo is too drop its licence down even further in permissibility to CC0 when it becomes even older and less usfull.

I hope this wall of text helps you at all and I'm sorry if it isnt the answer you wanted to hear.

TLDR: No, there is no command to do that but it wouldnt be too hard to make one and I would advise against it as this code is now old.
